From d0a549e8889ffce344a5b1f6c7b73525cff85dc7 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 3 Oct 2018 21:12:53 +0200
Subject: [PATCH 1/1] efi_loader: efi_connect_controller() use %pD

EFI_ENTRY in efi_connect_controller() should use %pD to print the remaining
device path.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_boottime.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/efi_loader/efi_boottime.c b/lib/efi_loader/efi_boottime.c
index fddfeb1a1a..faf3e43a51 100644
--- a/lib/efi_loader/efi_boottime.c
+++ b/lib/efi_loader/efi_boottime.c
@@ -2762,7 +2762,7 @@ static efi_status_t EFIAPI efi_connect_controller(
 	efi_status_t ret = EFI_NOT_FOUND;
 	struct efi_object *efiobj;
 
-	EFI_ENTRY("%p, %p, %p, %d", controller_handle, driver_image_handle,
+	EFI_ENTRY("%p, %p, %pD, %d", controller_handle, driver_image_handle,
 		  remain_device_path, recursive);
 
 	efiobj = efi_search_obj(controller_handle);
-- 
2.19.0

